apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -1170,19 +1142,16 @@ const SelectIsVirtual = ({value, readOnly, onChange}) => (
     style={readOnly ? styles.readOnlyInput : undefined}
     disabled={readOnly}
   >
-    <option key={'in_person'} value={'in_person'}>
+    <option key={false} value={false}>
       No, this is an in-person workshop.
     </option>
-    <option key={'friday_institute'} value={'friday_institute'}>
-      Yes, this is a Code.org-Friday Institute virtual workshop.
-    </option>
-    <option key={'regional'} value={'regional'}>
-      Yes, this is a regional virtual workshop.
+    <option key={true} value={true}>
```

Comment:
what will happen now if you look back at a workshop that had one of the previous choices set? Will it be blank or default to in-person or virtual?

Reply:
It'll use whether the workshop is virtual or not to decide what to show. I think looking at the old `currentVirtualStatus` method and where it was used before/after this change maybe can help clarify.

Any workshop that had selected the Friday Institute option was selecting a) `third_party_provider` = friday_institute, and b) `virtual` = true, so they'll see the "Yes, virtual" selection.

Reply:
> General question: what is our level of confidence that we won't bring back Friday Institute workshops next summer? If we think we might bring it back, it may be a good idea to keep most of the old code and hide it with a flag or something.

I asked Tawny this, and she said she had no reason to believe it'll come back. Not totally convinced we won't still be in a pandemic next summer and maybe still doing virtual stuff, but I think this PR could basically be reverted if we wanted to bring it back.